CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 - [Track requests and dependencies from ServiceBus .NET Client 2.1.0](https://github.com/Microsoft/ApplicationInsights-dotnet-server/issues/740)
 - [Fix for REST API Request filter bug](https://github.com/Microsoft/ApplicationInsights-dotnet-server/issues/175)
 - [Fix for SyntheticUserAgentTelemetryInitializer null check](https://github.com/Microsoft/ApplicationInsights-dotnet-server/issues/750)
+- [Track dependencies from EventHubs .NET Client 2.0.0](https://github.com/Azure/azure-event-hubs-dotnet/pull/203)
```

Comment:
Please link to the GH issue (https://github.com/Microsoft/ApplicationInsights-dotnet-server/issues/753).
Inside the issue, you can link to the eventHub PR .

Reply:
Done